### DISCUSSION TITLE
Item drag&drop functionality not working

### DISCUSSION THREAD

**Discussion by AnsariFurqan** [pnp/sp-dev-fx-webparts#2129] [Q&A]
Hello,
Thanks for the Kanban webpart,
All the functionality is working fine except item drag and drop functionality. When an item is selected & pressed and when try to drag it into another stack no action takes place even data is not getting updated. Please help me with the issue.

Regards,
Furqan

**Comment by hugoabernier** [comment]
Hey @AnsariFurqan. I would suggest that you open an issue  so that the authors get notified and possibly help you faster.
